Sources/ClientRuntime/Telemetry/DefaultTelemetry.swift

```diff
@@ -16,7 +16,7 @@ public enum DefaultTelemetry {
     public static let provider: TelemetryProvider = _DefaultTelemetryProvider()
 
     fileprivate class _DefaultTelemetryProvider: TelemetryProvider {
-        var contextManager: TelemetryContextManager { defaultTelemetryContextManager }
+        var contextManager: TelemetryContextManager { defaultContextManager }
```

Comment:
Q: What was the reason behind deleting `Telemetry` from the name?

Reply:
When referencing `defaultTelemetryContextManager` in the other files, it had a redundant name:

```swift
DefaultTelemetry.defaultTelemetryContextManager
```

Since it is already in the `DefaultTelemetry` enum, I think it makes sense to rename it to:

```swift
DefaultTelemetry.defaultContextManager
```

Reply:
Gotcha, that makes sense.